Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -640,17 +625,19 @@ def format_body(new_email_body):
     return html_body
 
 
-def single_thread_reply(email_code, incident_id, email_cc, add_cc, notes, attachments, files, email_subject,
+def single_thread_reply(email_code, incident_id, email_cc, add_cc, notes, body_type, attachments, files, email_subject,
                         subject_include_incident_id, email_to_str, service_mail, email_latest_message,
                         mail_sender_instance, reputation_calc_async=False):
     """
         Retrieve all entries in the EmailThreads context key
     Args:
+        reputation_calc_async:
```

Comment:
Why not at the end, like the argument itself, in the method's signature??
Also - If it's an empty description, why include it?